README.md

```diff
@@ -61,15 +61,18 @@ Read more about [Verify Pacts](https://github.com/realestate-com-au/pact/wiki/Ve
 ```js
 var pact = require('@pact-foundation/pact-node');
 var opts = {
-	providerBaseUrl: <String>,       // Running API provider host endpoint. Required.
-	pactUrls: <Array>,               // Array of local Pact file paths or Pact Broker URLs (http based). Required.
-	providerStatesUrl: <String>,     // URL to fetch the provider states for the given provider API. Optional.
-	providerStatesSetupUrl: <String>, // URL to send PUT requests to setup a given provider state. Optional.
-	pactBrokerUsername: <String>,    // Username for Pact Broker basic authentication. Optional
-	pactBrokerPassword: <String>,    // Password for Pact Broker basic authentication. Optional
+	providerBaseUrl: <String>,           // Running API provider host endpoint. Required.
+	pactBrokerUrl: <String>              // URL of the Pact Broker to retrieve pacts from. Required if not using pactUrls.
+	provider: <String>                   // Name of the Provider. Required.
+	tags: <Array>                        // Array of tags, used to filter pacts from the Broker. Optional.
```

Comment:
Hmmmm maybe comma separated and then split into an array?

But, does the CLI even work? Looking at `bin/pact-node` I can't see the `create()` method being referenced: `pact.create(program).start();`.

Running it results in:

```
matt ~/development/public/pact-node λ ./bin/pact-node
/Users/mfellows/development/public/pact-node/bin/pact-node:31
pact.create(program).start();
     ^

TypeError: pact.create is not a function
    at Object.<anonymous> (/Users/mfellows/development/public/pact-node/bin/pact-node:31:6)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:445:3
```

I have been a bit blind to its existence so thanks for reminding me.